apps/web/ee/lib/intercom/HelpMenuItemDynamic.tsx

```diff
@@ -1,8 +0,0 @@
-import dynamic from "next/dynamic";
-import { Fragment } from "react";
-
-const HelpMenuItemDynamic = process.env.NEXT_PUBLIC_INTERCOM_APP_ID
-  ? dynamic(() => import("./HelpMenuItem"))
-  : Fragment;
```

Comment:
The purpose of this is to prevent loading extra JS for self-hosted who don't use Intercom.

Reply:
We could do the same with Zendesk